rdrf/rdrf/services/io/defs/importer.py

```diff
@@ -1089,3 +1103,19 @@ def _create_cde_policies(self, registry_model):
                 cde_policy.save()
                 cde_policy.groups_allowed.set(groups)
                 cde_policy.save()
+
+    def _create_group_permissions(self, data):
+        from django.contrib.auth.models import Permission
+        from django.contrib.contenttypes.models import ContentType
+        for group_dict in data:
```

Comment:
There are a few other functions in `Importer` that clear the registry of objects before doing an import. I think it's worth doing the same here, as admins might expect importing a registry definition to delete any existing group permissions that aren't part of the new definition.

Reply:
As discussed, the `group.permissions.set(group_permissions)` will replace any existing permissions with those provided in the import so the absence of any existing group permissions will cause them to be deleted.